install/kubernetes/cilium/templates/cilium-ingress-service.yaml

```diff
@@ -24,6 +24,14 @@ spec:
     protocol: TCP
     nodePort: {{ .Values.ingressController.service.secureNodePort }}
   type: {{ .Values.ingressController.service.type }}
+  {{- if semverCompare ">=1.24-0" .Capabilities.KubeVersion.Version -}}
```

Comment:
these two fields are only applicable if ServiceType is LoadBalancer. Understand that the service type is added in dependently with this PR, so we can take care of this subsequently.

https://github.com/cilium/cilium/pull/22583